internal/core/tx.py

```diff
@@ -25,14 +28,61 @@ def tx_sign(unsigned_file_name, from_address, sequence, gas="auto"):
     Returns:
         _tuple_: (boolean, json|str)
     """
-    command = f"""{DAEMON} tx sign {HOME}/{unsigned_file_name} --from {from_address} \
+    if sequence is not None:
+        command = f"""{DAEMON} tx sign {HOME}/{unsigned_file_name} --from {from_address} \
 --chain-id {CHAINID} --keyring-backend test \
 --home {DAEMON_HOME}-1 --node {RPC} --signature-only=false \
 --sequence {sequence} --gas {gas} --output json"""
+    else:
+        command = f"""{DAEMON} tx sign {HOME}/{unsigned_file_name} --from {from_address} \
+--chain-id {CHAINID} --keyring-backend test \
+--home {DAEMON_HOME}-1 --node {RPC} --signature-only=false \
+--gas {gas} --output json"""
+
+    return exec_command(command)
+
+
+def tx_partner_sign(
```

Comment:
waht is this?

Reply:
@anil we are using this function. For multi sig purposes. Where each signer will use this function to generate their signs.